### ISSUE TITLE
Notifications: Add 'regex' and 'notregex' capability to the Notification Rules

### ISSUE THREAD

**Issue by kristinarobinson** [influxdata/ui#423]
**Proposal:**
From https://github.com/influxdata/EAR/issues/1932
Add 'regex' and 'notregex' to the notification rule's list of operators.

**Current behavior:**
Describe what currently happens.
Create a notification rule via the UI; add a tag condition; observe that you can only select == as the operator.
Use the REST API to retrieve the notificationRule definition
Use the REST API to PUT an update to the notificationRule, changing the operator on the tagRule from equal to 'regex'
Observe a success response from your PUT request.
Go back to the InfluxDB UI and try to view your modified notification rule
Get an error message: "An InfluxDB error has occurred. 
**
**Desired behavior:**
Create a notification rule via the UI; add a tag condition; be able to select the 'regex' operator';
Use the REST API to retrieve the notificationRule definition
Use the REST API to PUT an update to the notificationRule, changing the operator on the tagRule from equal to 'regex'
Observe a success response from your PUT request.
Go back to the InfluxDB UI and try to view your modified notification rule
View the modified notification rule sucessfully.

**Comment by barbaranelson** [comment]
No, this work is not yet prioritized. The notification UI is going through a redesign, so this will probably be addressed as part of that work, but we have no estimate yet for this work.

**Comment by danatinflux** [comment]
@barbaranelson Any updates on this one? It's been a while. :)

**Comment by barbaranelson** [comment]
No update - we added an alert UI in notebooks, that reduces the complexity we have around check/notifications/rules.  Is there a reason you're looking for an update now?

**Comment by danatinflux** [comment]
Going through our team's oldest FRs, trying to do some spring cleaning. :)
